### DISCUSSION TITLE
How to define complex domain?

### DISCUSSION THREAD

**Discussion by cylinch82** [lululxvi/deepxde#1264] [Q&A]
I have a complex domain $\Omega$ with its boundary like $\partial{\Omega}=(1 + cos^2{4 \theta}), \ \  r \leq 1, \ \  0 \leq \theta \leq 2\pi$ (it is defined in polar coordinates form).
I'm wondering that if there are suitable modules in `deepxde.geometry` for complex domain like polar coordinates (which Cartesian coordinates might be very complicated)?
I've tried the Pointcloud module but I can't define the domain by myself.

The domain looks like this
<img width="399" alt="截圖 2023-04-21 15 12 28" src="https://user-images.githubusercontent.com/42511817/233568650-ce1d126d-46e9-4938-88db-6988148798d5.png">

Thanks

**Comment by pescap** [comment]
`cos^2(4 \theta) = 1/2 [cos(8\theta) + 1]`. Therefore, the star-shaped parametrization in #1280 would be ok.
